src/test/java/com/example/amazoncom/BrowserStackSetup.java

```diff
@@ -0,0 +1,42 @@
+package com.example.amazoncom;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class BrowserStackSetup {
+  String driverPath = "C:\\geckodriver.exe";
+  WebDriver driver;
+  BrowserStackHomePage objBrowserStackHomePage;
+  BrowserStackSignUpPage objBrowserStackSignUpPage;
+  @BeforeTest
+  public void setup() {
+    System.setProperty("webdriver.chrome.driver",
+        "C:\\Users\\Julio_Morales\\chromedriver\\chromedriver.exe");
+    driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://www.amazon.es/");
```

Comment:
do not use driver actions when you use the PageObject pattern. all steps should be encapsulated inside the page object class. here should be the function call. i.e 
amazonSignInPage.openSignInPage().